kong/reports.lua

```diff
@@ -197,15 +208,15 @@ end
 -- `reset_counter` was set to 0 (with `kong_dict:set(key, 0)` we would lose the increment
 -- done by Worker B.
```

Comment:
It looks like the comment block above no longer belongs in this module, since the negative-incr module was moved to lua-resty-counter

Reply:
removed!